src/app/components/FlippableCard.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/prop-types */
```

Comment:
`PropTypes` are probably enabled by default for a reason. Please avoid lint overrides like this. If you think `PropTypes` should be disabled project-wide, open an issue and make your case there.